src/sciline/task_graph.py

```diff
@@ -79,3 +109,30 @@ def visualize(
         from .visualize import to_graphviz
 
         return to_graphviz(self._graph, **kwargs)
+
+    def _repr_html_(self) -> str:
+        leafs = sorted(
+            [
+                escape(keyname(key))
+                for key in (
+                    self._keys if isinstance(self._keys, tuple) else [self._keys]
+                )
+            ]
+        )
+        roots = sorted(
+            {
+                escape(keyname(key))
+                for key, (_, requires) in self._graph.items()
+                if len(requires) == 0
+            }
+        )
+        scheduler = escape(self._scheduler.__class__.__name__)
+        return ''.join(
+            (
+                '<p>Task graph computes</p>\n',
+                _list_items(leafs),
+                '<p>based on</p>\n',
+                _list_max_n_then_hide(roots),
+                f'<p>using the scheduler {scheduler}.</p>',
```

Comment:
I must say not a fan of the "narrative" formatting.  How about:

```
Output keys: A, B, C
Scheduler: DaskScheduler
Providers:
....
```
For the dask scheduler, it would be very helpful if we can include which underlying dask scheduler (`get` function) is used, see here: https://scipp.github.io/sciline/generated/classes/sciline.scheduler.DaskScheduler.html.

Reply:
It was a bit of an experiment ;) I'll change it.

However I don't think the last column should not be "Providers" but rather something like "Input parameters"?
Or do you think it would be better to list all providers in the task graph rather than only the inputs?

Once the user has selected what output they are interested in I think the intermediate providers are kind of uninteresting, while the inputs that contribute to a particular output might be interesting for debugging purposes / understanding what is needed to compute a particular quantitiy.

Reply:
> Once the user has selected what output they are interested in I think the intermediate providers are kind of uninteresting

Maybe? We can only list the inputs for now.